core/src/sidebar/sidebar.module.css

```diff
@@ -4,21 +4,14 @@
 	padding: 10px 8px;
 	display: flex;
 	flex-direction: column;
-	height: 100vh;
+	min-height: 100vh;
```

Comment:
What happens when I add a lot of items? Will the height go over 100vh?

Reply:
yes, the footer will be push down

Reply:
Push down is not a good experience since we'll have some menus pinned at the bottom of the sidebar.
What we want is enable overflow scrolling at the middle.